### DISCUSSION TITLE
screen mirroring/airplay

### DISCUSSION THREAD

**Discussion by bigoss** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#131] [Q&A] [answered]
My Alfa works with your drivers on OC Big Sur.  I had airplay with same usb card and your drivers on Catalina to watch things from computer on apple tv. Since Big Sur, screen mirroring does not work :-(  My card does not appear as Wi fi under system preferences / network . Is called 802.11n NIC instead. Is there some way to have the mirroring on apple tv with this card and os?

**Comment by chris1111** [comment, marked as answer]
I dont think so ?

**Comment by benkongau1** [comment]
I have same issue. How do I solve it please?
